website/common/locales/en/groups.json

```diff
@@ -270,5 +270,6 @@
   "confirmCancelGroupPlan": "Are you sure you want to cancel the group plan and remove its benefits from all members, including their free subscriptions?",
   "canceledGroupPlan": "Canceled Group Plan",
   "groupPlanCanceled": "Group Plan will become inactive on",
-  "purchasedGroupPlanPlanExtraMonths": "You have <%= months %> months of extra group plan credit."
+  "purchasedGroupPlanPlanExtraMonths": "You have <%= months %> months of extra group plan credit.",
+  "inviteLimitReached": "You have already sent the max number of email invites."
```

Comment:
Can this be changed to something like:

"You have already sent the maximum number of email invitations. We have a limit to prevent spamming, however if you would like more, please contact us at admin@habitica.com and we'll be happy to discuss it!"

The email address to use is `TECH_ASSISTANCE_EMAIL`

@lemoness Do you want different wording?